pom.xml

```diff
@@ -16,6 +16,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <camel.version>4.14.0-SNAPSHOT</camel.version>
         <langchain4j-version>1.2.0</langchain4j-version>
+        <langchain4j-beta-version>1.3.0-beta9</langchain4j-beta-version>
```

Comment:
Hm, I'm afraid of a conflict here. Would it be possible to use langchain4j 1.2.0 instead?

Reply:
For the langchain4j embedding stores, it's not possible because there is no 1.2.0 release yet (see https://mvnrepository.com/artifact/dev.langchain4j/langchain4j-weaviate for example).    Camel has the same issue in this area (and other areas of langchain4j) and deals with it by providing a langchain4j-version and langchain4j-beta-version 

https://github.com/apache/camel/blob/main/parent/pom.xml#L338-L341

I think we may have to do the same thing here, and then resolve it once there's a full 1.2.0 release